tests/platform_tests/api/test_sfp.py

```diff
@@ -689,8 +689,15 @@ def test_lpmode(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, pla
                 logger.warning("test_lpmode: Skipping transceiver {} (not applicable for this transceiver type)".format(i))
                 continue
 
+            lpmode_state_pretest = sfp.get_lpmode(platform_api_conn, i)
+            if lpmode_state_pretest is None:
```

Comment:
@longhuan-cisco we should fail the test if None. get_lpmode() is not optional.

Reply:
I was just following the original behavior just same as original code of [#L696](https://github.com/sonic-net/sonic-mgmt/blob/34ce7e3c2cec20a7619af5b005dea21282bed8ef/tests/platform_tests/api/test_sfp.py#L696) to just throw warning and do ```break```. Shall we not follow it, and change the original behavior at [#L696](https://github.com/sonic-net/sonic-mgmt/blob/34ce7e3c2cec20a7619af5b005dea21282bed8ef/tests/platform_tests/api/test_sfp.py#L696) as well (not sure if this will break existing test result on some platform)?

Reply:
Ok. looks like some platforms don't support LPmode